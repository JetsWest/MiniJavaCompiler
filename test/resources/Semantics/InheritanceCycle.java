class SomeClass {
    public static void main(String[] args) {
        System.out.println(0);
    }
}

class A extends B { }

class B extends C { }

class C extends A { }