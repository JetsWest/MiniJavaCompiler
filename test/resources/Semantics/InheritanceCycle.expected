(Line 7) Cycle detected in dependency graph. Check your class extensions.